integration/src/main/resources/servers/workspace-alpha.json

```diff
@@ -11,5 +11,18 @@
   "testRunnerServiceAccountFile": "testrunner-sa.json",
 
   "skipDeployment": true,
-  "skipKubernetes": true
-}
\ No newline at end of file
+  "skipKubernetes": true,
+
+  "versionScripts": [
+    {
+      "name": "ReadFromTerraHelmfileRepo",
+      "description": "Version from https://github.com/broadinstitute/terra-helmfile",
+      "parameters": ["workspacemanager","terra-helmfile/versions/app/alpha.yaml", "terra-helmfile/environments/live/alpha.yaml"]
+    },
+    {
+      "name": "ReadFromGitCommitLog",
+      "description": "Version from Git Commit Log",
```

Comment:
version of what exactly?

Reply:
I have updated the description to `Hash of git branch from Git Commit Log`.